config/routes.rb

```diff
@@ -177,6 +177,8 @@
       resources :watched_words, only: [:index, :create, :update, :destroy] do
         collection do
           get "action/:id" => "watched_words#index"
+          get "action/:id/download" => "watched_words#download"
+          delete "action/:id/clear_all" => "watched_words#clear_all"
```

Comment:
We haven't been great at enforcing it but we prefer urls to use dashes instead of underscores. The ruby method can be `clear_all` but the URL would ideally be `clear-all`